mysql-paas-replication/azuredeploy.json

```diff
@@ -0,0 +1,228 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serverName": {
+            "type": "string",
+            "defaultValue": "uniqueString",

```

Comment:
this needs to be globally unique, e.g.

"defaultValue": "[concat('sql', uniqueString(resourceGroup().id)]",

Reply:
Done